src/views/GitWorkflow.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <workflow-layout />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import WorkflowLayout from "@/components/Workflow/WorkflowLayout.vue";
```

Comment:
@JuanJosePG  workflow* Please review the inport or the name from the filters, cause in my pc there are problems with the folders name